Assets/SteamVR_Unity_Toolkit/Scripts/VRTK_InteractableObject.cs

```diff
@@ -28,7 +28,8 @@ public enum GrabAttachType
             Spring_Joint,
             Track_Object,
             Rotator_Track,
-            Child_Of_Controller
+            Child_Of_Controller,
+            Handhold
```

Comment:
Is Handhold the best word to describe the mechanic?

Would it be something like `Climbable` or something like that?